app/src/main/java/com/example/snaplapse/MapFragment.kt

```diff
@@ -1,17 +1,25 @@
 package com.example.snaplapse
 
+import android.location.Location
 import android.os.Bundle
 import androidx.fragment.app.Fragment
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
+import com.google.android.gms.location.FusedLocationProviderClient
+import com.google.android.gms.location.LocationServices
 import com.google.android.gms.maps.GoogleMap
 import com.google.android.gms.maps.OnMapReadyCallback
 import com.google.android.gms.maps.SupportMapFragment
 import com.google.android.gms.maps.model.LatLng
 import com.google.android.gms.maps.model.MarkerOptions
 
 class MapFragment : Fragment(), OnMapReadyCallback {
+
+    private lateinit var mMap: GoogleMap
+    private lateinit var client: FusedLocationProviderClient
+    private lateinit var activity: MainActivity
```

Comment:
can we remove client and activity vars for now, since theyre not being used

Reply:
or comment out

Reply:
yeah sure